src/ui/public/doc_table/components/table_row.js

```diff
@@ -83,6 +85,16 @@ module.directive('kbnTableRow', function ($compile) {
         createSummaryRow($scope.row, $scope.row._id);
       });
 
+      $scope.getContextAppHref = () => {
+        return `#/context/${$scope.indexPattern.id}/${$scope.row._type}/${$scope.row._id}?${
+          $httpParamSerializer({
+            _a: rison.encode({
+              columns: $scope.columns,
+            }),
```

Comment:
This is a good point. I have switched to using `KbnUrl`, but as you hinted at, Kibana breaks on many levels when the index pattern id contains non-ascii characters. I was unable to properly configure it in management nor to load anything in discover, so testing it is a bit difficult.